src/web/components/ApiKeyManagement/ApiRolesCell.tsx

```diff
@@ -0,0 +1,26 @@
+import { ApiRoleDTO } from '../../../api/entities/ApiRole';
+import { Tooltip } from '../Core/Tooltip';
+
+import './ApiRolesCell.scss';
+
+type ApiRolesProps = {
+  apiRoles: ApiRoleDTO[];
+  showRole?: Boolean;
```

Comment:
I find this prop a bit unclear.
1. Can we perhaps give it a clearer name, perhaps `showRoleNameToolTip`
2. We can use `boolean` instead of `Boolean` here - the primitive version is preferred here

Reply:
Have renamed